src/components/TodoItem.tsx

```diff
@@ -29,4 +41,4 @@ export function TodoItem({ index, item, handleTodoItemChecked }: TodoItemProps)
         <button onClick={handleDeleteClick}>Delete</button>
       </div>
     );
-  }
+    }
```

Comment:
This `}` got indented, and should be moved out a tabs worth.

replace
```
    );
    }
```
with
```
    );
}
```

Reply:
handled in [77230de](https://github.com/ZachWitzig/todoist/pull/3/commits/77230de0aa48039c04a2d2707327e22d807c5848)